<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>2</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDial" name="kiloDial">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-color: rgb(85, 0, 0);</string>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="frequencyLcd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="kiloLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kHz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hundredLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 100 Hz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="hundredDial">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-color: rgb(85, 0, 0);</string>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>9</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="tenDial">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-color: rgb(85, 0, 0);</string>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>9</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tenLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 10 Hz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDial" name="oneDial">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>150</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-color: rgb(85, 0, 0);</string>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>9</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="oneLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="durationSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="kiloLcd">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hundredLcd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tenLcd">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="oneLcd">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="durationLcd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="playSound">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Play Sound</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duration (s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSound_Generator">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="action1_kHz"/>
    <addaction name="action100_Hz"/>
    <addaction name="action666_Hz"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionToiminto1"/>
    <addaction name="actionToiminto2"/>
   </widget>
   <addaction name="menuSound_Generator"/>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action1_kHz">
   <property name="text">
    <string>1 kHz</string>
   </property>
  </action>
  <action name="action100_Hz">
   <property name="text">
    <string>100 Hz</string>
   </property>
  </action>
  <action name="action666_Hz">
   <property name="text">
    <string>666 Hz</string>
   </property>
  </action>
  <action name="actionToiminto1">
   <property name="text">
    <string>Toiminto1</string>
   </property>
  </action>
  <action name="actionToiminto2">
   <property name="text">
    <string>Toiminto2</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>durationSlider</tabstop>
  <tabstop>kiloDial</tabstop>
  <tabstop>hundredDial</tabstop>
  <tabstop>tenDial</tabstop>
  <tabstop>oneDial</tabstop>
  <tabstop>playSound</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kiloDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kiloLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hundredDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hundredLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tenDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tenLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>oneLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>durationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>durationLcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
